File: 10 - Python/02 - Estrutura Condicional/Exercício 12.py
#Exercício - Tempo de Jogo

horaInicial = int(input("Digite a hora inicial do jogo: "))
horaFinal = int(input("Digite a hora final do jogo: "))

if horaInicial < horaFinal:
    duracao = horaFinal - horaInicial
else:
    duracao = (24 - horaInicial) + horaFinal

print(f"O jogo durou {duracao} hora(s).")

File: 06 - C/Exercícios/02 - Estrutura Condicional/Exercício 12.c
//Exercício - Tempo de Jogo

#include <stdio.h>

int main() {

    int horaInicial, horaFinal, duracao;

    printf("Hora inicial: ");
    scanf("%d", &horaInicial);
    
    printf("Hora final: ");
    scanf("%d", &horaFinal);

    if(horaInicial < horaFinal){
        duracao = horaFinal - horaInicial;
    }
    else{
        duracao = (24 - horaInicial) + horaFinal;
    }

    printf("O jogo durou %d hora(s)", duracao);

    return 0;

}